Index: src/test.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>public class test {\r\n    public static void main(String[] args){\r\n<<<<<<< HEAD\r\n        System.out.println(\"Hello\");\r\n        System.out.println(\"HELLO INI MARYAM NAFISAHH BARUU\" );\r\n=======\r\n\r\n\r\n        System.out.println(\"Hello HII DINI HAH\");\r\n>>>>>>> 6c8d1e26718b8e6555a8ad6d4a55475c95282cda\r\n    }\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/test.java b/src/test.java
--- a/src/test.java	(revision 77c9bb239e32f304e86ae06c7b9a89a4b7bc72e8)
+++ b/src/test.java	(date 1751648764786)
@@ -1,12 +1,4 @@
 public class test {
     public static void main(String[] args){
-<<<<<<< HEAD
-        System.out.println("Hello");
-        System.out.println("HELLO INI MARYAM NAFISAHH BARUU" );
-=======
 
-
-        System.out.println("Hello HII DINI HAH");
->>>>>>> 6c8d1e26718b8e6555a8ad6d4a55475c95282cda
-    }
 }
